lib/spack/spack/test/spec_semantics.py

```diff
@@ -314,6 +314,8 @@ def test_satisfies_multi_value_variant(self):
         check_satisfies('multivalue-variant foo="bar,baz"',
                         'foo="bar"')
 
+    @pytest.mark.skipif(sys.platform == 'win32',
+                        reason="Not supported on Windows (yet)")
     def test_satisfies_single_valued_variant(self):
```

Comment:
Like tests for concretization, most of these tests confirm behavior that is fundamental, so ideally we can get as many of them running as possible.

Reply:
One small error I'm chasing down.

Reply:
Now passing.